src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
I would model `BoolOption` after the same entity in `boost::program_options`, i.e. give it ability to parse supplied value and set an implicit one, which is taken when no explicit value is provided.
If  you, @briankendall, do not want to do that or wait for such implementation, IMO `StringOption` with a custom check would be fine. Let's just avoid option name change in the future.